src/ArchiveAction.ts

```diff
@@ -32,7 +33,7 @@ export default abstract class ArchiveAction {
   }
 
   static prepareAction(action: ArchiveAction) {
-    verbose(core.getBooleanInput('verbose'));
```

Comment:
It's really not clear what `getBooleanInput` does, but it didn't seem to work as expected.